hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -1750,6 +1753,67 @@ public void testUnbuffer() throws IOException {
 
   }
 
+  /**
+   *  Test EC checksum with Replicated checksum.
+   */
+  @Test
+  public void testFileChecksum() throws IOException {
```

Comment:
Can we make this test to have parameterized and run with different lengths of files? ex: partial stripes and multiple blocks, single chunk, multichunk with partial stripe and padding etc?

Reply:
Yes definitely, Will add the test coverage for all the scenarios.